docs/pages/install/docker.mdx

```diff
@@ -82,3 +89,52 @@ CONTAINER ID   IMAGE  COMMAND   CREATED  STATUS   PORTS    NAMES
 # docker stop <container-name>, which is `pelican-director` as previously defined
 docker stop pelican-director
 ```
+
+## Configure Pelican server in a container
+
+There are two ways to configure a Pelican server running in a container. One is through the environment variables, the other is by passing a `pelican.yaml` configuration file to the container. This section includes instructions for both.
+
+### Configure via environment variables
+
+All Pelican configuration parameters described in [parameters page](../parameters.mdx) can be passed to a Pelican server as environment variables. For a configuration parameter, the corresponding environment variable is `PELICAN_<PARAMETER_NAME>`, where `PELICAN` is the prefix, and `<PARAMETER_NAME>` is the name of the parameter, with dots `.` replaced by underscores `_`. For example, a Pelican configuration named `Logging.Level` has a corresponding environment variable named `PELICAN_LOGGING_LEVEL`.
```

Comment:
This actually isn't quite true, especially for lists of objects like our `Origin.Exports` block. Maybe just say "Most Pelican configuration parameters... "

Reply:
Sounds good. I'll leave a note about the `object` type params